frontend/src/libs/hooks/use-get-current-user/use-get-current-user.hook.ts

```diff
@@ -0,0 +1,24 @@
+import { LocalStorage, StorageKey } from '~/libs/packages/storage/storage.js';
+import { actions as authActions } from '~/slices/auth/auth.js';
+
+import { useAppDispatch } from '../hooks.js';
```

Comment:
```suggestion
import { useAppDispatch } from '../use-app-dispatch/use-app-dispatch.hook.js';
```
because useGetCurrentUser exported from the same hooks.js file